src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.cpp

```diff
@@ -455,4 +455,9 @@ bool emberAfSmokeCoAlarmClusterSelfTestRequestCallback(CommandHandler * commandO
     return SmokeCoAlarmServer::Instance().HandleRemoteSelfTestRequest(commandObj, commandPath);
 }
 
+bool __attribute__((weak)) emberAfPluginSmokeCoAlarmSelfTestRequestCommand(EndpointId endpointId)
```

Comment:
Shouldn't this be left un-implemented so apps that include this cluster realize they need to implement this?

As in: this should be implemented in all-clusters-app, not here.

Reply:
I added a `smco-stub.cpp` file to implement the `emberAfPluginSmokeCoAlarmSelfTestRequestCommand` function. Is it okay to do this?